src/v1/base/Stepper.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  Children,
+  createContext,
+  PropsWithChildren,
+  ReactNode,
+  useContext,
+} from 'react';
+
+import { IconCheckStroke } from '@public/icons';
+import ProgressBar from '@/v1/base/ProgressBar';
+
+type StepStatus = 'complete' | 'incomplete' | 'active';
+
+type StepContextValues = {
+  index: number;
+  status: StepStatus;
+  count: number;
+};
+
+const StepperContext = createContext<StepContextValues>(
+  {} as StepContextValues
+);
+
+const Stepper = ({
+  activeIndex,
+  children,
+}: PropsWithChildren<{ activeIndex: number }>) => {
+  const stepElements = Children.toArray(children);
+  const stepCount = stepElements.length;
+
+  const progressPercent =
+    activeIndex === 0 ? 0 : Math.ceil((activeIndex / (stepCount - 1)) * 100);
+
+  const getStepStatus = (step: number): StepStatus => {
+    if (step < activeIndex) return 'complete';
+    if (step > activeIndex) return 'incomplete';
+    return 'active';
+  };
+
+  return (
+    <div className="relative z-[1] flex w-full items-center justify-between">
+      <ProgressBar value={progressPercent} className="-z-[1]" />
+      {stepElements.map((child, index) => (
+        <StepperContext.Provider
+          key={index}
+          value={{ index, status: getStepStatus(index), count: stepCount }}
+        >
+          {child}
+        </StepperContext.Provider>
+      ))}
+    </div>
+  );
+};
+
+const getStepClasses = (status: StepStatus) => {
+  switch (status) {
+    case 'complete':
+      return 'bg-main-900';
+    case 'active':
+      return 'bg-main-900 w-[7.4rem] delay-[400]';
+    case 'incomplete':
+    default:
+      return 'bg-main-500';
+  }
+};
+
+const Step = ({
+  label,
+  children,
+}: {
+  label?: string;
+  children?: ReactNode;
+}) => {
+  const { status, index } = useContext(StepperContext);
+  const statusClasses = getStepClasses(status);
+  const activeAnimationClasses =
+    index !== 0 ? 'opacity-0 animate-stepper-transition' : 'opacity-1';
+
+  const stepNumberToRender = index + 1;
+  const labelToRender = label ? label : stepNumberToRender;
+
+  return (
+    <div
+      className={`relative flex h-[3rem] w-[3rem] shrink-0 flex-col items-center justify-center rounded-full duration-500 ${statusClasses} overflow-hidden`}
+    >
+      {status === 'complete' ? (
+        <IconCheckStroke className="h-auto w-[1rem]" />
+      ) : status === 'active' ? (
+        <p
+          className={`relative self-baseline whitespace-nowrap px-[1.2rem] py-[0.4rem] text-white font-body2-bold ${activeAnimationClasses}`}
+        >
+          {labelToRender}
+        </p>
+      ) : (
+        <p className="relaive text-white font-body2-bold">
```

Comment:
p1;

```suggestion
        <p className="relative text-white font-body2-bold">
```

오탈자가 있군요..!
수정이후 relative가 적용되었을 때와 아닐 때 의도대로 잘 동작하는지 확인이 필요할 것 같아요!!